charts/tfy-gpu-operator/values.yaml

```diff
@@ -455,6 +455,29 @@ azure-aks-gpu-operator:
         requests:
           cpu: 10m
           memory: 200Mi
+      ## @skip azure-aks-gpu-operator.node-feature-discovery.master.tolerations
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Equal
+          effect: NoSchedule
+        - key: node-role.kubernetes.io/control-plane
+          operator: Equal
+          effect: NoSchedule
+        - key: node.kubernetes.io/not-ready
+          operator: Exists
+          effect: NoExecute
+          tolerationSeconds: 300
+        - key: node.kubernetes.io/unreachable
+          operator: Exists
+          effect: NoExecute
+          tolerationSeconds: 300
+        - key: node.kubernetes.io/memory-pressure
+          operator: Exists
+          effect: NoSchedule
```

Comment:
We should not add these

Reply:
these are there in chart: like these are default values. I can remove